internal/indexmigration/migration.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package indexmigration
+
+import (
+	"os"
+
+	"k8s.io/klog"
+
+	"sigs.k8s.io/krew/internal/environment"
+	"sigs.k8s.io/krew/internal/gitutil"
+	"sigs.k8s.io/krew/pkg/constants"
+)
+
+// Done checks if the krew installation requires a migration to support multiple indexes.
+// A migration is necessary when the index directory doesn't contain a "default" directory.
+func Done(paths environment.Paths) (bool, error) {
+	f, err := os.Stat(paths.DefaultIndexPath())
```

Comment:
I think this is not a good check.
$KREW_ROOT/index/default not existing might also mean someone just installed krew but there is no default index.

I propose **$KREW_ROOT/index/.git** exists (doesn't even have to be a dir) is a quite good check.

Reply:
That's a good idea. I'll make the change.